deploy/deploy.yaml

```diff
@@ -146,6 +146,8 @@ spec:
                         - name
                         type: object
                       type: array
+                    failedNodesReason:
```

Comment:
This commit seems to have a mix of "pure rename" and "additional stuff". If it's not too much work, you may want to split the two using good old `git add -p` ;-)

Reply:
yes, I can do that

Reply:
done :-)

Reply:
done